client/extensions/woocommerce/app/shipping/shipping-labels.js

```diff
@@ -24,21 +24,50 @@ class ShippingLabels extends Component {
 		//TODO: use redux state with real data
 		this.state = {
 			visible: true,
-			cardData: {
+			cards: [ {
+				selected: true,
 				type: 'VISA',
 				digits: '1234',
 				name: 'Name Surname',
 				date: '12/19'
-			}
+			}, {
+				selected: false,
+				type: 'MasterCard',
+				digits: '5678',
+				name: 'Name Surname',
+				date: '01/20'
+			} ]
 		};
 
 		this.onToggle = this.onToggle.bind( this );
+		this.renderCard = this.renderCard.bind( this );
 	}
 
 	onToggle() {
 		this.setState( { visible: ! this.state.visible } );
 	}
 
+	selectCard( index ) {
+		const cards = this.state.cards.map( ( card ) => {
+			return { ...card, selected: false };
+		} );
+
+		cards[ index ].selected = true;
+
+		this.setState( { cards } );
+	}
```

Comment:
I think in future "UI only" PRs stuff like this can be omitted - the card selection should come from the redux state tree (and this is essentially reducer code).